.github/actions/dockerhub/Dockerfile.debian-source

```diff
@@ -0,0 +1,21 @@
+FROM rust:bullseye as build
+
+ARG STACKS_NODE_VERSION="No Version Info"
+ARG GIT_BRANCH='No Branch Info'
+ARG GIT_COMMIT='No Commit Info'
+ARG BUILD_DIR=/build
+ARG TARGET=x86_64-unknown-linux-gnu
+WORKDIR /src
+
+COPY . .
+
```

Comment:
I think this is missing:
```
RUN apt-get update && apt-get install -y git
```
since this line is in the [Dockerfile.linux-glibc-x64](https://github.com/stacks-network/stacks-blockchain/pull/3199/files#diff-a915f90fa919dda92131d8945ab78933a02ea43d7e3d373347cac02c4c045789) build script.

Reply:
Technically, neither are needed but it also doesn't hurt so i'll add it

Reply:
https://github.com/wileyj/stacks-blockchain/commit/d8ce51e8d9a4990728bc43acf5e48c80a8079103